tuf/store/filestore.go

```diff
@@ -43,6 +43,10 @@ type FilesystemStore struct {
 func (f *FilesystemStore) GetMeta(name string, size int64) ([]byte, error) {
 	fileName := fmt.Sprintf("%s.%s", name, f.metaExtension)
 	path := filepath.Join(f.metaDir, fileName)
+	if _, err := os.Stat(path); os.IsNotExist(err) {
```

Comment:
No need to do the `Stat`, can just check the `err` from `ioutils.ReadFile` to see if it's `IsNotExists`